lib/lambda_ethereum_consensus/store/block_db.ex

```diff
@@ -76,6 +104,16 @@ defmodule LambdaEthereumConsensus.Store.BlockDb do
     Logger.info("[BlockDb] Pruning finished. #{Enum.count(slots_to_remove)} blocks removed.")
   end
 
+  defp decode_block_info(root, data) do
+    with {:d, {encoded_signed_block, status}} <- {:d, :erlang.binary_to_term(data)},
```

Comment:
Sorry, could you explain me the purpose of the :d tag here? I don't catch it.

Reply:
Long story short, to identify the line, but it seems like it's a controversial technique. Forum on the topic: https://elixirforum.com/t/proper-way-of-handling-simple-booleans-in-with-statement/45556/20

I'll change it to something more readable.